### DISCUSSION TITLE
Vision Peer-Review Comments

### DISCUSSION THREAD

**Discussion by open-schnick** [FileFighter/filefighter.github.io#4] [General]
Hey,
comment your feedback about our "Vision" page from your peer-review here:

**Comment by MeNoSmartBrain** [comment]
* Link to other domains by opening a new tab (comment section)
* NAS = Network Attached Storage and is not a "cloud"
* Check for grammatical errors (example: first sentence on "our vision")
* Opensource = exploitable is a harsh statement to make
* by the way: no security through obscurity
* I really enjoyed your jokes about dying and studying
* got --> get the opportunity
* got --> go to about.
* , to --> . To get some more
* code does not "go"
* fansboys --> fanboys
*  don't use the word "themeable"
* don't link to stuff that doesn't exist

Your feature set sounds promising. I'd like to know your project will be implemented. Will you provide users with an application, an OS image or solutions like docker. Furthermore, it is unclear to me how you what to enable/ support throughput at the gateway of home users and the availability of a NAS through the internet. Generally speaking, it will be difficult to implement a NAS as a cloud service.  
In general, I really like your idea and wish you all the best, with your project. I hope from the bottom of my heart that you don't advice inexperienced users to make self-installed computers accessible through the internet ... (Pls don't)

Kind regards,

The Vaultionizer Team, TINF19B2

**Comment by open-schnick** [comment]
Hello dear Vaultionizer Team,
Thank you for your very useful feedback.
We are still in the progress of developing our technical architecture and general structure. You will be able to read about these kind of things in our next blog, comming up next week.

**Comment by open-schnick** [comment]
> * don't link to stuff that doesn't exist
>
Small feedback from our side, would be nice to know which link you mean.
:)

**Comment by nf3lix** [comment]
Hi FileFighter-Team,

After reading your first blog post as well as your vision, we think that your project will be really helpful in fighting google and other companies that collect data.

What we like:

- the idea of protecting the users privacy
- layout of your blog
- about page with infos about the individual team members
- design of your blog and especially the logo

What you can improve:
- “We will short talk about” → “We will briefly talk about”
- Try to convince the reader that they need this product in the very first sentence! 

Furthermore, the security aspect was unfortunately only mentioned in one sentence.
Do you already know what exactly will be done to protect the user’s data from unauthorized access? For instance, how do you plan to store the user passwords? 
Beyond that, will it be possible to recover one’s login data in case of loss?

We’re looking forward to your future blog post and your finished project!

Good coding! 
Plandora-Team

**Comment by qvalentin** [comment]
@nf3lix 
Dear Plandora-Team,
thanks for your Feedback.
All the data will be stored in secured databases, and at least the passwords should also be encrypted.
The admin of each instance of our software will be responsible for managing their users, but the details of how this will work are not defined yet.
